### DISCUSSION TITLE
Dask Load balancer - Allow users to add a list of servers for ML/Hardware Transcoding/SIMD stuff[Feature] <feature-name-goes-here>

### DISCUSSION THREAD

**Discussion by Donovoi** [immich-app/immich#9234] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Create another container/server that is a nginx/dask load balancer. this should allow users to add as many servers as they like to run GPU/SIMD tasks.

most likely a big job, but I would hope it helps with processing where the communication between the load balancer, user, and worker is not terrible.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Feel free to put a loadbalancer in your setup, there are a few people who do. I don't think we need to include this by default though.
